extract/extract.py

```diff
@@ -40,42 +44,42 @@ class Extract(ServiceBase):
     MAX_EXTRACT_LIVE = 100
 
     LAUNCHABLE_EXTENSIONS = [
-        '.ade',
-        '.adp',
-        '.as',   # Adobe ActionScript
-        '.bat',  # DOS/Windows batch file
-        '.chm',
-        '.cmd',  # Windows command
-        '.com',  # DOS command
-        '.cpl',
-        '.exe',  # DOS/Windows executable
-        '.dll',  # Windows library
-        '.hta',
-        '.inf',  # Windows autorun file
-        '.ins',
-        '.isp',
-        '.jar',  # Java JAR
-        '.jse',
-        '.js',   # Javascript
-        '.lib',
-        '.lnk',  # Windows shortcut
-        '.mde',
-        '.msc',
-        '.msp',
-        '.mst',
-        '.pif',
-        '.py',   # Python script
-        '.scr',  # Windows screen saver
-        '.sct',
-        '.shb',
-        '.sys',
-        '.vb',   # VB Script
-        '.vbe',  # Encrypted VB script
-        '.vbs',  # VB Script
-        '.vxd',
-        '.wsc',
-        '.wsf',
-        '.wsh'
+        ".ade",
+        ".adp",
+        ".as",  # Adobe ActionScript
+        ".bat",  # DOS/Windows batch file
+        ".chm",
+        ".cmd",  # Windows command
+        ".com",  # DOS command
+        ".cpl",
+        ".exe",  # DOS/Windows executable
+        ".dll",  # Windows library
+        ".hta",
+        ".inf",  # Windows autorun file
+        ".ins",
+        ".isp",
+        ".jar",  # Java JAR
+        ".jse",
+        ".js",  # Javascript
+        ".lib",
+        ".lnk",  # Windows shortcut
+        ".mde",
+        ".msc",
+        ".msp",
+        ".mst",
+        ".pif",
+        ".py",  # Python script
+        ".scr",  # Windows screen saver
+        ".sct",
+        ".shb",
+        ".sys",
+        ".vb",  # VB Script
+        ".vbe",  # Encrypted VB script
+        ".vbs",  # VB Script
+        ".vxd",
+        ".wsc",
+        ".wsf",
+        ".wsh",
```

Comment:
Why are all the single quote strings replaced with double quote strings?

Reply:
Because I'm using the Black formatter with very few overriding configurations.

> Black prefers double quotes (" and """) over single quotes (' and '''). It will replace the latter with the former as long as it does not result in more backslash escapes than before.
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html

Reply:
Oh nice.